### PR TITLE
Added support RPC state_get_entity

### PR DESCRIPTION
**Pull request by ZhmakAS**
Implement new RPC method get_state_entity

* Resolves: # 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits are signed
- [ ] Tests included/updated or not needed
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by koltsov-iv — APPROVED** [review]
LGTM!

File: rpc/rpc_client.go [hunk 1]
```diff
@@ -127,6 +127,42 @@ func (c *client) QueryGlobalStateByStateHash(ctx context.Context, stateRootHash
...
+	resp, err := c.processRequest(ctx, MethodGetStateEntity, ParamGetStateEntity{EntityIdentifier: entityIdentifier}, &result)
+	if err != nil {
+		return StateGetEntity{}, err
+	}
+
+	result.rawJSON = resp.Result
+	return result, nil
+}
+
+func (c *client) GetEntityByHash(ctx context.Context, entityIdentifier EntityIdentifier, hash string) (StateGetEntity, error) {
```
Comment (davidatwhiletrue) [inline rpc/rpc_client.go:142]: GetEntityByBlockHash

File: rpc/rpc_client.go [hunk 2]
```diff
@@ -127,6 +127,42 @@ func (c *client) QueryGlobalStateByStateHash(ctx context.Context, stateRootHash
...
+	resp, err := c.processRequest(ctx, MethodGetStateEntity, ParamGetStateEntity{EntityIdentifier: entityIdentifier, BlockIdentifier: &BlockIdentifier{Hash: &hash}}, &result)
+	if err != nil {
+		return StateGetEntity{}, err
+	}
+
+	result.rawJSON = resp.Result
+	return result, nil
+}
+
+func (c *client) GetEntityByHeight(ctx context.Context, entityIdentifier EntityIdentifier, height uint64) (StateGetEntity, error) {
```
Comment (davidatwhiletrue) [inline rpc/rpc_client.go:154]: GetEntityByBlockHeight

File: rpc/request.go [hunk 3]
```diff
@@ -97,11 +98,26 @@ type ParamGetAccountInfoBalance struct {
...
+	BlockIdentifier  *BlockIdentifier `json:"block_identifier,omitempty"`
+}
+
 type AccountIdentifier struct {
 	AccountHash *key.AccountHash
 	PublicKey   *keypair.PublicKey
 }
 
+// EntityIdentifier Identifier of an addressable entity.
+type EntityIdentifier struct {
```
Comment (davidatwhiletrue) [inline rpc/request.go:112]: Add:
NewEntityIdentifierFromAccountHash
NewEntityIdentifierFromPublicKey
NewEntityIdentifierFromEntityAddr

**Review by davidatwhiletrue — COMMENTED** [review]
@ZhmakAS, added couple of comments.

**Pull request merged**